proto/internal/buf.yaml

```diff
@@ -1,12 +1,18 @@
 ---
 version: "v1"
 deps:
-  - "buf.build/beta/protoc-gen-validate"
+  - "buf.build/envoyproxy/protoc-gen-validate:bb405eae115246f0b5ccf8997136e3d8"
   - "buf.build/authzed/api"
 lint:
   except:
     - "ENUM_VALUE_PREFIX"
     - "ENUM_ZERO_VALUE_SUFFIX"
+  ignore_only:
+    FIELD_LOWER_SNAKE_CASE:
+      - "core/v1/core.proto" # TODO: Remove after v0 api removed
 breaking:
   use:
     - "WIRE"
+  ignore_only:
+    FIELD_WIRE_COMPATIBLE_TYPE:
+      - "dispatch/v1/dispatch.proto" # TODO: Remove after core v1 is merged
```

Comment:
This ignore allows the conversion from v0 to core in the dispatch api to be merged